src/client.ts

```diff
@@ -51,6 +53,30 @@ export class NuklaiCoreApiClient {
   }
 }
 
+export interface ActionResult {
```

Comment:
I'm a little confused about this. This wouldn't apply to all the results though for different actions though

Reply:
Right! Extended it